packages/app-utils/src/components/AuthBootstrap.tsx

```diff
@@ -48,7 +48,10 @@ export function AuthBootstrap({ children }: AuthBootstrapProps): JSX.Element {
       let isCanceled = false;
       async function loadAuthConfigValues(): Promise<void> {
         try {
-          const newAuthConfigValues = await client.getAuthConfigValues();
+          const newAuthConfigValues = (await client.getAuthConfigValues()) as [
+            string,
+            string,
+          ][];
```

Comment:
This kind of sucks, but I don't have any clever solutions. Would need a type predicate to map the `string[][]` to `[string, string][]` by checking each element I guess? `isMapConstructorArray` or something.
I also assume that GWT can't express this typing, otherwise that would be the best option to fix it in the jsapi-types source (though can have this temporarily).

Reply:
I did note this in a non-critical improvements ticket https://github.com/deephaven/deephaven-core/issues/5082

Ya we could add a util to actually check it since this is just assuming it's an array of string pairs